setup-env/src/index.js

```diff
@@ -0,0 +1,20 @@
+const core = require('@actions/core');
+const ActionInput = require('./actionInput');
+
+/**
+ * Entry point to initiate the Action.
+ * 1. Triggers parsing of action input values
+ * 2. Decides requirement of Local Binary
```

Comment:
Points 2,3,4 are not happening in this action. Please update the comments.

Reply:
Will update them.

Reply:
done